x-pack/plugins/data_enhanced/common/search/session/types.ts

```diff
@@ -5,6 +5,9 @@
  */
 
 export interface BackgroundSessionSavedObjectAttributes {
+  realmType?: string;
+  realmName?: string;
+  username?: string;
```

Comment:
@legrego as you mentioned some security work being done in this area, are we going in the right direction here?

Reply:
There's a good chance that we will be producing a unique user identifier that doesn't require Kibana to rely on `realmType` + `realmName` to disambiguate the `username`. I'm trying to think through a recommendation for you in the short term that won't collide with our future state - I'll report back here soon